src/app/shared/entities/location/location-actions.ts

```diff
@@ -4,35 +4,29 @@ import { Location } from '@app/types';
 import { LocationEntity } from './location-state';
 import { MapEntity } from '../map/map-state';
 
-export enum Events {
```

Comment:
With the NGRX change that added the `ofType` filter really don't need this Action Label block, so moved the label to be defined with the Action and added a `Key` property that will help capture groups of actions for metareducers